FWCore/Utilities/src/InputTag.cc

```diff
@@ -83,9 +83,9 @@ namespace edm {
   }
 
   InputTag::InputTag(InputTag&& other)
-      : label_(std::move(other.label())),
```

Comment:
Neither is what what intended. The move needs to be done using the member data not the member functions.